sample-apps/s3-java/README.md

```diff
@@ -75,7 +79,12 @@ To invoke the function directly, run `5-invoke.sh`.
         "ExecutedVersion": "$LATEST"
     }
 
-Let the script invoke the function a few times and then press `CRTL+C` to exit.
+Let the script invoke the function a few times and then press `CRTL+C` to exit. Note that you
```

Comment:
I would increase the timeout to 30 seconds in the CloudFormation templates (`template.yml`, `template-mvn.yml`) to avoid timeouts and the need for this note.

Reply:
Done.